### DISCUSSION TITLE
✅ Replace select control with choose control for Align Content control in container widget

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#22640] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

For a better user experience, it would be great if you replace the current Align Content control with a choose control in the container widget.

### Describe the solution you'd like

![container](https://github.com/elementor/elementor/assets/77163248/000d02f4-82a4-4c83-8865-2b3acdf03032)

**Comment by rami-elementor** [comment]
Fix: https://github.com/elementor/elementor/pull/29149

**Comment by nicholaszein** [comment]
Hello again @79ho3ein!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.26.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
